test/keys/rsa.spec.js

```diff
@@ -88,7 +88,7 @@ describe('RSA', function () {
 
   describe('export and import', () => {
     it('password protected PKCS #8', async () => {
-      const pem = await key.export('pkcs-8', 'my secret')
+      const pem = await key.export('my secret', 'pkcs-8')
```

Comment:
Is it worth adding a test without an explicit format?

Reply:
There is one :) It's on line 98

Reply:
oh ok, didn't realize it!